app/configurator/components/annotation-options.tsx

```diff
@@ -92,6 +103,14 @@ const AnnotationOptions = (props: AnnotationOptionsProps) => {
               />
             </Box>
           ))}
+          <Button
+            onClick={handleClosePanel}
+            sx={{ alignSelf: "flex-end", minHeight: 32, mt: 3, px: 5 }}
```

Comment:
Nit: Why do we have to override minHeight and px, this should be handled via the button right ?

Reply:
Thanks for catching, for height I could use size="small" 👍 For px, it looks different for different buttons in the design currently, see e.g. [this button](https://www.figma.com/file/Hn9wvuEYLUmdJtg4KwZYUx/visualize.admin.ch---design?type=design&node-id=3718-229434&mode=design&t=MBXaeZBQkyiFi3RM-4) and [this button](https://www.figma.com/file/Hn9wvuEYLUmdJtg4KwZYUx/visualize.admin.ch---design?type=design&node-id=3718-229077&mode=design&t=MBXaeZBQkyiFi3RM-4) – in fact maybe this should be minWidth applied in the theme, but I didn't want to do a full review of design to find a rule in this PR, maybe we could check with @KerstinFaye?

Reply:
[Here are again all Button sizes in the library](https://www.figma.com/file/kYsVD11OnsPAcDGYCHz5jH/Federal-Projects-Global-Library-%E2%80%93%C2%A0CURRENT?type=design&node-id=97-8822&mode=design&t=FYQj7douDXJw5OXL-4)
In this case I would apply the small button size.